flexget/plugins/sites/site1337x.py

```diff
@@ -11,10 +11,12 @@
 from flexget.utils.requests import RequestException
```

Comment:
Ahh. The standard is actually different in pep8 for module names though, I think it's better with the underscore in. https://www.python.org/dev/peps/pep-0008/#package-and-module-names (and paranoidi is also the one who named it with the underscore :wink:)